PP_APP/server/frontend/static/js/script.js

```diff
@@ -1,23 +1,35 @@
+function isEmail(email) {
+    var regex = /^([a-zA-Z0-9_.+-])+\@(([a-zA-Z0-9-])+\.)+([a-zA-Z0-9]{2,4})+$/;
+    return regex.test(email);
+}
+
 $(document).ready(function () {
     $("#btnSubmit").click(function (event) {
         
         //stop submit the form, we will post it manually.
         event.preventDefault();
 
-        // var dumb_ob = JSON.parse('{ "response": { "img_post_id": 23, "packager": "DummyBrand", "infer_img": "/media/infer/Image_from_iOS.jpg", "outerbox": 30344, "innerbox": 11739, "item": null }}')
+        // var dumb_ob = JSON.parse('{ "response": { "img_post_id": 23, "packager": "DummyBrand", "infer_img": "/media/infer/Image_from_iOS.jpg", "outer_size": 30344, "inner_size": 11739, "item_size": null }}')
         // alert(dumb_ob.response.infer_img)
         // Get form
         var form = $('#fileUploadForm')[0];
 
         // Create an FormData object 
         var data = new FormData();
 
+        var user_email = document.getElementById('user_email');
         var top_input = document.getElementById('top_img');
         var side_input = document.getElementById('side_img');
         // If you want to add an extra field for the FormData
         data.append("packager", "DummyBrand");
+        data.append("email", user_email.value);
         data.append("top_img", top_input.files[0]);
         data.append("side_img", side_input.files[0]);
+
+        if (!isEmail(user_email.value)){
+            $("#status").text('Please enter valid email address.');
+            return
+        }
```

Comment:
> I was able to run this code on my computer, but I did not set up tensorflow. So did this PR address the issues we were having before of running this Django app on the Macs?

From what I know, using this app on MAC has two issues.
1) Setting up the MASK R-CNN API. Look at the README for more detail.
2) The requirements.txt does not match up with the one on Github. This is because I am using Linux and the server is using Linux. So there is some cross-compatibility issue. We should create a separate requirements.txt for MAC users.